include/seccomp.h.in

```diff
@@ -216,6 +216,12 @@ struct scmp_arg_cmp {
 #endif /* AUDIT_ARCH_RISCV64 */
 #define SCMP_ARCH_RISCV64	AUDIT_ARCH_RISCV64
 
+/**
+ * The SuperH SH architecture tokens
+ */
```

Comment:
@drakenclimber The const defines ```AUDIT_ARCH_SHEL``` and ```AUDIT_ARCH_SH``` are defined by the kernel. So we have to stick to those names. I did switch ```SCMP_``` in the seccomp namespace since little-endian is the common flavor of SH.

Reply:
@glaubitz would it be odd for the SH folks if we used it "shbe" instead of "sheb"?  I ask because it seems to match "Big Endian" better and we already have "ppc64le" using the "[b,l]e" format.

Reply:
GCC matches both, so we can also use ```shbe``` if you prefer that.

> https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=gcc/config.gcc;h=7276be31019a8742d7bb1b056418e1490bd72bb5;hb=a1fcb9a1debf225f87ffd4d0a29bbc24ba3a6689#l2170

Reply:
I remember now why I used ```eb```. It's because the kernel uses the terminology as well.

Reply:
FWIW, the little-endian MIPS port is called ```mipsel64```, so that naming scheme is already being used. I think we should stick to what the kernel uses.

Reply:
> I think we should stick to what the kernel uses.

Agreed, that makes sense to me, thanks!